cloudinit/util.py

```diff
@@ -3231,8 +3229,8 @@ def deprecate(
 
     Note: uses keyword-only arguments to improve legibility
     """
-    if not hasattr(deprecate, "_log"):
-        deprecate._log = set()  # type: ignore
+    if not hasattr(deprecate, "log"):
```

Comment:
Looking at this again, I don't think we want this (or the test updates based on it).

See the first comment on https://stackoverflow.com/a/16955098/37205